auth0/src/main/java/com/auth0/android/provider/OAuthManager.java

```diff
@@ -104,6 +98,16 @@ void startAuthentication(Activity activity, String redirectUri, int requestCode)
     @SuppressWarnings("ConstantConditions")
     @Override
     boolean resume(AuthorizeResult result) {
+        // Should never happen where the algorithms required are not present, but just in case it does,
+        // this will allow us to at least call the error handler callback with an exception including
+        // the cause
+        try {
+            PKCE.throwIfNotAvailable();
+        } catch (Throwable t) {
+            callback.onFailure(new AuthenticationException("Algorithims required to use PKCE are not available on this device. Authentication can not be done securely.", t));
```

Comment:
typo. Also, is this change unit tested?

```suggestion
            callback.onFailure(new AuthenticationException("Algorithms required to use PKCE are not available on this device. Authentication can not be done securely.", t));
```